### ISSUE TITLE
darknet模型转换后，利用ncnn2mem转成二进制加载时会报:layer -1 not exists or registered

### ISSUE THREAD

**Issue by ChisenZhang** [Tencent/ncnn#959]
darknet 模型，转成ncnn模型后直接调用没问题；利用ncnn2mem 转成二进制的之后加载（net.load_param_bin）就会报：layer -1 not exists or registered 用net.load_param加载就会报param is too old, please regenerate的错误？ 什么原因？看了darknet2ncnn里面的代码param文件里面的标记是7767517呀。

**Comment by bryanlinnan** [comment]
请问你这个问题解决了吗

**Comment by ChisenZhang** [comment]
模型长这样

[net]
#Training
#batch=16
#subdivisions=4

Testing
batch=1
subdivisions=1

height=256
width=256
channels=3
# min_crop=128
# max_crop=256

burn_in=1000
learning_rate=0.0005 # 初始学习率
policy=poly
power=4
max_batches=800000 # 最大bath
momentum=0.9 # 动量
decay=0.0005 # 权重衰减正则项

angle=7 # 旋转角度
hue=.1 # 色调
saturation=.75 # 饱和度
exposure=.75 # 曝光量
#aspect=.75 #  

adam=1
B1=0.9
B2=0.999
eps=1e-7

[convolutional]
filters=16
size=7
stride=4
pad=1
batch_normalize=1
activation=leaky

#S1
[convolutional]
filters=32
size=1
stride=1
pad=1
batch_normalize=1
activation=leaky

#DWConv
[convolutional]
filters=32
size=3
stride=2
groups=32
pad=1
batch_normalize=1
activation=leaky

[convolutional]
filters=16
size=1
stride=1
pad=1
batch_normalize=1
activation=linear

#S2
[convolutional]
filters=32
size=3
stride=2
pad=1
batch_normalize=1
activation=leaky

#DWConv
[convolutional]
filters=64
size=1
stride=1
pad=1
batch_normalize=1
activation=leaky

[avgpool]

[convolutional]
filters=2
size=1
stride=1
pad=1
activation=linear

[softmax]
groups=1

**Comment by ChisenZhang** [comment]
> 请问你这个问题解决了吗

没有，还没有找到原因

**Comment by bryanlinnan** [comment]
请问你有试过其他模型利用ncnn2mem，然后运行吗？我这边可以正确加载，但是运行会报错

**Comment by ChisenZhang** [comment]
> 请问你有试过其他模型利用ncnn2mem，然后运行吗？我这边可以正确加载，但是运行会报错

没有试过，有时间试一下demo
